### PR TITLE
make local-deploy

### PR DESCRIPTION
**Pull request by guicassolato**
Make target to ease rebuilding and redeploying Authorino to the local Kubernetes cluster

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -145,3 +145,8 @@ endif
 .PHONY: local-setup
 local-setup: kustomize kind
 	utils/local-setup.sh
+
+# Rebuild and push the docker image and redeploy authorino to the local k8s cluster
+.PHONY: local-deploy
```
Comment (guicassolato) [inline Makefile:150]: Happy to rename it in case you think it's confusing with `local-setup`.

File: utils/local-deploy.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -euo pipefail
+
+export AUTHORINO_NAMESPACE="authorino"
+export KIND_CLUSTER_NAME="authorino-integration"
+
+echo "Building Authorino"
+docker build -t authorino:devel ./
+kind load docker-image authorino:devel --name ${KIND_CLUSTER_NAME}
```
Comment (guicassolato) [inline utils/local-deploy.sh:10]: I know. Some DRY opportunity here: https://github.com/3scale-labs/authorino/blob/3cae60bd3305ec98e1d1f3c4664c040d0c731042/utils/local-setup.sh#L10-L12

File: Makefile (hunk 1 above)
Comment (didierofrivia) [reply Makefile:150]: Let's review both tasks after some use

**Comment by didierofrivia** [comment]
You might want to add the task description to readme? maybe we could include a separate Development.md file

File: Makefile (hunk 1 above)
Comment (guicassolato) [reply Makefile:150]: `make local-image-reload`?

Comment (guicassolato) [reply Makefile:150]: `make local-image`?

Comment (jmprusi) [reply Makefile:150]: what about `make build-to-kind` ? 

This script will end up, building the images and loading them into the local Kind cluster, not only making images.

Comment (guicassolato) [reply Makefile:150]: > what about make build-to-kind ?

I think better keeping it consistent with the other one, `local-setup`. It seems that "local" already means "to kind".

**Review by didierofrivia — APPROVED** [review]
LGTM, let's merge to aid development, and then we can add steps to the readme along with any improvement

**Pull request merged**